src/sentry/migrations/0171_backfill_actors.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 1.11.29 on 2021-03-05 18:22
+
+from django.db import migrations
+from sentry.models import Actor
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_null_actors(apps, schema_editor):
+    User = apps.get_model("sentry", "User")
+    Team = apps.get_model("sentry", "Team")
+
+    for user in RangeQuerySetWrapperWithProgressBar(User.objects.filter(actor=None)):
+        user.actor = Actor.objects.create(type=1)
+        user.save()
+
+    for team in RangeQuerySetWrapperWithProgressBar(Team.objects.filter(actor=None)):
+        team.actor = Actor.objects.create(type=0)
+        team.save()
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = True
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
```

Comment:
Have to set this to `False`. Otherwise, we'll have one very large transaction that ends up locking more and more of the table.